quickstart/monitoring/monitoring-quickstart.ipynb

```diff
@@ -101,7 +94,7 @@
    "id": "51eb099a",
    "metadata": {},
    "source": [
-    "You can download the TruEra Python SDK via [PyPI](https://pypi.org/project/truera/) with `pip install truera`.\n",
+    "You can download the TruEra Python SDK via [PyPI](https://pypi.org/project/truera/) with `pip install truera==12.6.0rc0`.\n",
```

Comment:
This quickstart will need an update after each truera package update, that is a dependency.

Reply:
Fixed this